src/utilities.js

```diff
@@ -0,0 +1,4 @@
+export const formatName = (string) => {
+  const split = string.split("_");
+  return split.map(word => word[0].toUpperCase() + word.substr(1)).join(" ");
```

Comment:
The second part is cool. I haven't used `substr` yet! Although remember that everything is already set to all upper case. If you want proper casing, you should edit it in the CSS using 'text-transform: capitalize.'

Reply:
That's a much cleaner way, I always forget about text-transform... Thank you for the reminder 😄